src/index.html

```diff
@@ -1,17 +1,184 @@
+        <legend>Registration</legend>
+
+        <div class="section">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+          >
+        </div>
+        <div class="section">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            maxlength="10"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting facts about you</legend>
+        <div class="section">
+          <label for="radio">Do you love cats?</label>
+          <input
+            type="radio"
+            name="radio"
+            id="yes"
+            value="Yes"
+          >
+          <label for="yes">
+            Yes
+          </label>
+          <input
+            type="radio"
+            name="radio"
+            id="no"
+            value="No"
+          >
+          <label for="no">No</label>
+        </div>
+        <div class="section">
+          <label for="favcolor">What is your favorite color?</label>
+          <input
+            type="color"
+            name="favcolor"
+            id="favcolor"
+          >
+        </div>
+        <div class="section">
+          <label for="bedtime">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="bedtime"
+            id="bedtime"
+            min="1"
+            max="100"
+          >
+        </div>
+        <div class="section">
+          <label for="cars">What are your favorite brand of car?</label>
+          <select
+            name="cars"
+            id="cars"
+            multiple
+          >
+            <option value="empty"></option>
+            <option value="Audi">Audi</option>
+            <option value="BMW">BMW</option>
+            <option value="LADA">LADA</option>
+          </select>
+        </div>
+        <div class="section">
+          <label for="rate">How do you rate our work?</label>
+          <input
+            type="range"
+            name="rate"
+            id="rate"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          Additional info
+        </legend>
+        <div class="section">
+          <label for="text">Comments:</label>
+          <textarea
+            name="text"
+            id="text"
+            rows="2"
+            cols="20"
+            autocomplete="off"
+          >
+          </textarea>
+        </div>
+        <label for="answer">Would you recommend us?</label>
+        <select
+          name="answer"
+          id="answer"
+        >
+          <option value="yes">yes</option>
+          <option value="no">no</option>
+        </select>
+      </fieldset>
+      <input type="submit"
+             value="Submit"
+      >
```

Comment:
before pushing to github, don't forget to check your code style, it improves your professionalism

```suggestion
      <input 
        type="submit"
        value="Submit"
      >
```

OR (if there are less than 3 attributes)

```suggestion
      <input  type="submit"  value="Submit">
```